Sources/GRPC/AsyncAwaitSupport/PassthroughMessageSource.swift

```diff
@@ -29,7 +29,7 @@ import NIOCore
 /// indicate that the sequence should end with an error.
 @available(macOS 10.15, iOS 13, tvOS 13, watchOS 6, *)
 @usableFromInline
-internal final class PassthroughMessageSource<Element, Failure: Error> {
+internal final class PassthroughMessageSource<Element: Sendable, Failure: Error> {
```

Comment:
I don't think, we should enforce this.

Reply:
I'm curious as to why not. A non-`Sendable` `AsyncSequence` is next to useless: you can only send to it and receive from it on the same `Task`. When is anyone ever going to do that?